src/assets/CurrencyRateController.ts

```diff
@@ -195,16 +195,25 @@ export class CurrencyRateController extends BaseController<
       pendingCurrentCurrency,
       pendingNativeCurrency,
     } = this.state;
+
+    let conversionDate: number = Date.now() / 1000;
```

Comment:
Nit: Given that we're now setting this here (which makes perfect sense to me), maybe we don't need to set it in the return for `fetchExchangeRate` anymore? It seems silly to calculate this the same way twice.

Reply:
Agreed